middleware/github.js

```diff
@@ -67,15 +69,15 @@ async function setupTeams(req, token, client = null) {
 async function getTeams(client, org) {
   try {
     const resp = await client.users.getTeams();
-    return resp.data
-      .filter(entry => entry.organization.login === org)
-      .map(entry => entry.name);
+    return resp.data.filter(entry => entry.organization.login === org).map(entry => entry.name);
```

Comment:
Here it actually put it all on one line for us 👍

Reply:
yeah, Would love to understand the inconsistency.  On the one hand, the first one above was structurally simpler yet ended up on multiple lines. This one has two lambdas yet is all on one line...

Reply:
i think there may be a 3-chain then break rule, I'll have to look it up tomorrow. May just be a suggestion someone made.

Reply:
Ah yes it was merged here https://github.com/prettier/prettier/issues/1565

And via the comments it seems like this should be broken up b/c it's a 3-chain call. So I played with it locally and it *is* broken up if you limit the chars to 80, in fact it's the same code as the original. But at 120 it stays on 1 line, so it seems that this rule only applies after the char limit is reached.